Tabloid/Models/Tag.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
```

Comment:
Don't need to include System.Runtime.InteropServices or System.Runtime.CompilerServices here